packages/app/src/cli/services/deploy.ts

```diff
@@ -119,6 +125,66 @@ export async function deploy(options: DeployOptions) {
         },
       ]
 
+      if (partnersApp.betas?.declarativeWebhooks) {
```

Comment:
I think it will definitely be too quick to read the messages, so agree it makes sense to keep under the main deploy task. We'll pull the code out eventually though once things are finalized. The versioned config getting update is still an atomic action of an app deploy